windows/win_user.ps1

```diff
@@ -101,7 +101,8 @@ If ($account_locked -ne $null) {
 
 $groups = Get-Attr $params "groups" $null
 If ($groups -ne $null) {
-    If ($groups.GetType().Name -eq "String") {
+    $groups_type = $groups.GetType().Name
+    If ($groups_type -eq "String[]" -or $groups_type -eq "String") {
         [string[]]$groups = $groups.Split(",")
```

Comment:
I haven't looked at the rest of this but will this line work if $groups_type is "String[]" or "String"?.
If $groups_type can only be "String[]" then the -or $groups_type -eq "String" is not needed perhaps?